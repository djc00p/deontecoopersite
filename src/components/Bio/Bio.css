.bio {
}
a {
  color: #ffffff;
  text-decoration: none;
}
.bio-p {
  background-color: #901010;
  border-top: 0.25rem solid black;
  border-bottom: 0.25rem solid black;
  padding: 0.5rem 10%;
  font-size: 1.4rem;
}
