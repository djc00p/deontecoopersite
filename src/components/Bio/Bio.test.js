import React from "react";
import { shallow, mount, render } from "enzyme";
import Bio from "./Bio";

describe("Header", () => {
  it("should create snapshot of header", () => {
    const wrapper = shallow(<Bio />);
    expect(wrapper).toMatchSnapshot();
  });
});
