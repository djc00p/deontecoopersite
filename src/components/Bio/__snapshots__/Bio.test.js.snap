// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should create snapshot of header 1`] = `
<div
  className="My-Bio"
>
  <section
    className="bio"
  >
    <p>
      My name is Deonte J. Cooper. I was born in Denver, CO but have lived most of my days in Aurora, CO. I am a Software Engineer that studied at 
      <a
        href="https://turing.io/"
        target="_blank"
      >
        Turing School of Software and Design
      </a>
       in the 1901 Backend Engineer program where 
      <a
        href="https://rubyonrails.org/"
        target="_blank"
      >
        Ruby on Rails
      </a>
       is primarily taught 6/7 months and the last month was spent learning 
      <a
        href="https://www.javascript.com/"
        target="_blank"
      >
        JavaScript
      </a>
      . In those seven months, I learned Object Oriented Programming in Ruby, building Web Applications in Ruby, building Professional Rails applications, and Client-Side Development with JavaScript. I enjoy both frontend and backend web development. I have worked with other programming languages such as 
      <a
        href="https://reactjs.org/"
        target="_blank"
      >
        React
      </a>
       and 
      <a
        href="https://www.python.org/"
        target="_blank"
      >
        Python
      </a>
      .
    </p>
    <p>
      Some hobbies of mine are Video Games and Music Production. I have been playing video games since as far back as I can remember and that is what sparked my interest in programming. I mostly play RPGs (Role-Playing Games) like The Witcher 3, Mass Effect, Pokemon, Dark Souls and many more. I also enjoy FPS (First-Person Shooters) games like Call of Duty and Battlefield. I enjoy a well-told story and a good challenge. Dark Souls is the gaming version of how I describe programming. It is challenging but not impossible, with a little grit and understanding of how things work you can overcome any challenge. Music is my favorite creation by humankind. It is one of the few things that can bring people of different backgrounds together and have an impact on each individual. It can envelop emotions that can speak the truth of the heart without one even knowing and there is always a song for someone out there. Music is my life and passion and I want to share that passion with the world.
    </p>
    <p>
      I am currently learning about 
      <a
        href="https://en.wikipedia.org/wiki/Hydroponics"
        target="_blank"
      >
        Hydroponics Gardening
      </a>
       so that I can grow fruits and vegetables in my home without the use of soil. In hopes of using what I've learned, I would like to help my family and friends start growing at home as well. I would like to start an mini side business with my excess produce and sell it at farmers marts. I would also like to share what I've learned with others to show that anyone can do it as well. I also have a love and passion for reptiles, so I'm looking into getting into reptile keeping and possibly one-day breeding.
    </p>
  </section>
</div>
`;
